### PR TITLE
Add definitions for utf8 module

### PR DESCRIPTION
**Pull request by nickzelei**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: utf8/utf8.d.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Decodes any given UTF-8-encoded string (byteString) as UTF-8, and returns the UTF-8-decoded version of the string.
+     * It throws an error when malformed UTF-8 is detected.
+     * @param {string} byteString
+     */
+    decode(byteString: string): string;
+}
+
+declare var Utf8: Utf8;
```
Comment (vvakame) [inline utf8/utf8.d.ts]: Is this right? https://github.com/mathiasbynens/utf8.js#api

Comment (zelein) [inline utf8/utf8.d.ts]: Are you asking if the utf8 module is a static object?

**Comment by zelein** [comment]
I believe this is ready to go.
Unsure as to why the DT Bot has not fully marked its checkboxes as this build passes the Travis Test and is the correct naming convention.

**Comment by yuit** [comment]
@zelein thanks for the PR !

**Pull request merged**